biotracks/pushtopandas.py

```diff
@@ -6,33 +6,32 @@
     from datapackage.pushpull import _convert_path as convert_path
 import pandas as pd
 
+from .utils import NullLogger, get_logger
 from .names import OBJECTS_TABLE_NAME, LINKS_TABLE_NAME
 
 
-def push_to_pandas(directory, object_id_cmso):
-    """Push the datapackage to a pandas storage.
+def push_to_pandas(directory, object_id_cmso, log_level=None):
+    """Push a datapackage to a pandas storage.
 
-    Keyword arguments:
-    directory -- the directory to look into for the json descriptor
-    object_id_cmso -- the join_id specified in the .ini file
+    arguments:
+    directory -- the datapackage directory containing the json file
+    object_id_cmso -- the object id
     """
+    if log_level is None:
```

Comment:
Does this need to be special cased here or should `log_level=None` be handled by `get_logger` ?